docs/csharp/index.yml

```
@@ -0,0 +1,107 @@
+### YamlMime:Landing
+
+title: "C# programming language documentation"
+summary: "Learn how to write any application using the C# programming language on the .NET platform."
+
+metadata:
+  title: "C# programming language documentation - get started, tutorials, reference."
```

```suggestion
  title: "C# docs - get started, tutorials, reference."
```

I think metadata titles are supposed to be short for SEO.